gensrc/thrift/FrontendService.thrift

```diff
@@ -324,6 +324,14 @@ struct TTableStatus {
     5: optional i64 last_check_time
     6: optional i64 create_time
     20: optional string ddl_sql
+    // id for materialized view
+    21: optional string id
+    // database_name for materialized view
+    22: optional string database_name
```

Comment:
for information_schema.tables and information_schema.views, thet both needn't database_name in thrift, but could get database from  them, so I think we needn't this field.

Reply:
the field here is for sql like 'SHOW MATERIALIZED VIEW WHERE 1=1;' , and it is same with the show mv return field